Assets/SEE/Scanner/TokenMetrics.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace SEE.Scanner
+{
+    /// <summary>
+    /// Provides metrics calculated based on code tokens.
+    /// </summary>
+    public static class TokenMetrics
```

Comment:
Please provide test cases for your metrics.

Reply:
I assume I can just add them into `TestGraphProviders`? Or should I create a new test class?